sdk/cosmos/azure-cosmos/src/samples/java/com/azure/cosmos/ChangeFeedProcessorLatestVersionChangesCodeSnippet.java

```diff
@@ -0,0 +1,56 @@
+package com.azure.cosmos;
```

Comment:
missing license header.

Reply:
As discussed offline, I have removed all the public API changes, so this file is also removed